.drone.star

```diff
@@ -3535,3 +3536,91 @@ def onlyofficeService():
             ],
         },
     ]
+
+def deployments(ctx):
+    result = {
+        "kind": "pipeline",
+        "type": "docker",
+        "name": "k3d",
+        "privileged": True,
+        "steps": wait(ctx) + install(ctx) + showPodsAfterInstall(ctx),
+        "services": [
+            {
+                "name": "k3d",
+                "image": "ghcr.io/k3d-io/k3d:5-dind",
```

Comment:
```suggestion
                "image": "ghcr.io/k3d-io/k3d:5-dind",
                "privileged": True,
```
Doesn't the privileged flag need to go to service configuration?
See working example from ocis-charts repo: https://github.com/owncloud/ocis-charts/blob/main/.drone.star#L158-L170

Reply:
yeah, true

Reply:
Updated the privileged config but still same error:
https://drone.owncloud.com/owncloud/ocis/45082
>linter: untrusted repositories cannot enable privileged mode

Reply:
I believe this is something DevOps can enable.

Drone privileged mode: https://docs.drone.io/pipeline/docker/syntax/steps/#:~:text=Privileged%20Mode,access%20to%20your%20host%20machine.

Possible solution: https://docs.drone.io/cli/repo/drone-repo-update/